CDS/scripts/atom_image.pl

```diff
@@ -70,12 +73,31 @@ sub is_imageurl_valid {
 
 my $capi = decode_json($response->content);
 
-unless (is_imageurl_valid($capi->{'response'}->{'media'}->{'data'}->{'media'}->{'trailImage'}->{'master'}->{'file'})) {
-	exit(1);
+if(defined $ENV{'output-smaller-than'}){
+  my %image_urls;
+  my $image_place = 0;
+  while($image_place < 20) {
```

Comment:
is there any reason for the specific value of 20 here? if it's limited for a specific reason then a for loop https://perlmaven.com/for-loop-in-perl might be more appropriate

Reply:
Having viewed various output from CAPI it seems unlikely that there will be more than nineteen entries to sort. More than ten is quite unlikely, put the extra values are in there just in case. The code checks if a value is present on each pass and does nothing if nothing is found.

Reply:
just scan every item in the list. that way there is no arbitary limit

Reply:
The new code does this.